### ISSUE TITLE
Unit test give an error on the test graph builder

### ISSUE THREAD

**Issue by ne0z** [bridgecrewio/checkov#3256]
**Describe the issue**
Hello, I tried follow the [Contributing](https://github.com/bridgecrewio/checkov/blob/master/CONTRIBUTING.md) instruction page. And stuck on the "Running tests" section. The problem when I run pytest, it always shows an error on the test_graph_builder.py. So is it only an error on my laptop?

**Examples**
```bash
$ pipenv run python -m coverage run -m pytest tests/graph/terraform/graph_builder/test_graph_builder.py             
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
======================================================== test session starts =========================================================
platform darwin -- Python 3.7.13, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: /Users/danangheriyadi/Work/ne0z/checkov
collected 6 items                                                                                                                    

tests/graph/terraform/graph_builder/test_graph_builder.py ....F.                                                               [100%]

============================================================== FAILURES ==============================================================
___________________________________ TestGraphBuilder.test_build_graph_with_linked_registry_modules ___________________________________

self = <tests.graph.terraform.graph_builder.test_graph_builder.TestGraphBuilder testMethod=test_build_graph_with_linked_registry_modules>

    def test_build_graph_with_linked_registry_modules(self):
        resources_dir = os.path.realpath(
            os.path.join(TEST_DIRNAME, '../resources/modules/registry_security_group_inner_module'))
    
        graph_manager = GraphManager(NetworkxConnector())
        local_graph, tf_definitions = graph_manager.build_graph_from_source_directory(resources_dir,
                                                                                      render_variables=True,
                                                                                      download_external_modules=True)
    
        outputs_vpcs = self.get_vertex_by_name_and_type(local_graph, BlockType.OUTPUT, 'this_security_group_vpc_id',
>                                                       multiple=True)

tests/graph/terraform/graph_builder/test_graph_builder.py:135: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

local_graph = <checkov.terraform.graph_builder.local_graph.LocalGraph object at 0x112009a10>
block_type = <BlockType.OUTPUT: 'output'>, name = 'this_security_group_vpc_id', multiple = True

    @staticmethod
    def get_vertex_by_name_and_type(local_graph, block_type, name, multiple=False):
>       vertices = [local_graph.vertices[i] for i in local_graph.vertices_block_name_map[block_type][name]]
E       KeyError: 'this_security_group_vpc_id'

tests/graph/terraform/graph_builder/test_graph_builder.py:69: KeyError
====================================================== short test summary info =======================================================
FAILED tests/graph/terraform/graph_builder/test_graph_builder.py::TestGraphBuilder::test_build_graph_with_linked_registry_modules
==================================================== 1 failed, 5 passed in 13.12s ====================================================
```

**Version (please complete the following information):**
 - Checkov Version [master]

**Comment by ne0z** [comment]
After pulling the latest changes, it solves my problem.
